patches/components-version_info-BUILD.gn.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/version_info/BUILD.gn b/components/version_info/BUILD.gn
+index 8fbad36091b4a1e1baea40911599ac0d2d12aa0a..6b92b22b6bfc688dfe36219133f41b6431a54348 100644
+--- a/components/version_info/BUILD.gn
++++ b/components/version_info/BUILD.gn
+@@ -11,6 +11,7 @@ declare_args() {
+ }
+ 
+ static_library("version_info") {
++  configs += [ "//brave/build:version" ]
```

Comment:
this was unfortunately the only reasonable way to avoid a full rebuild on chromium version changes. I think the patch is worth the lost dev time of putting it in the top-level compiler config